keps/sig-storage/695-skip-permission-change/README.md

```diff
@@ -0,0 +1,287 @@
+
+## Production Readiness Review Questionnaire
+
+### Feature enablement and rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate
+    - Feature gate name: ConfigurableFSGroupPolicy
+    - Components depending on the feature gate: kubelet, kube-apiserver
+
+* **Does enabling the feature change any default behavior?**
+  No enabling the feature gate does not change any default behaviour.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+  the enablement)?**
+  Yes the feature-gate can be disabled once enabled. This will cause all volume mounts that
+  require permission change to use recursive permission and ownership change. This is the default
+  behaviour without the feature gate.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  For pods that have expected value in `pod.Spec.SecurityContext.FSGroupChangePolicy` as defined in this KEP,
+  it will start using specified policy.
+
+* **Are there any tests for feature enablement/disablement?**
+  There aren't any e2e but there are unit tests that cover this behaviour.
+
+### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?**
+  Since this feature requires users to opt-in by setting new field in pod spec, it should
+  not impact already running workloads.
+
+* **What specific metrics should inform a rollback?**
+  If after enabling this feature and setting `pod.Spec.SecurityContext.FSGroupChangePolicy`
+  users notice an increase in volume mount time via `storage_operation_duration_seconds{operation_name=volume_mount}`
+  or an increase in mount error count via `storage_operation_errors_total{operation_name=volume_mount}`
+  then a cluster-admin may want to rollback the feature.
+
+* **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
+  We have not fully tested upgrade and rollback. We have unit tests that cover the scenario
+  of feature gate being enabled and then disabled. But we will need to do more upgrade->downgrade->upgrade testing.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features,
+  APIs, fields of API types, flags, etc.?**
+  This feature deprecates no existing functionality.
+
+
+### Monitoring requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
+  Operator can query `pod.Spec.SecurityContext.fsGroupChangepolicy` field and identify if this is being set to non-default values.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to
+  determine the health of the service?**
+  - [x] Metrics
+    - mount operation duration:
+        - Metric name: storage_operation_duration_seconds{operation_name=volume_mount}
```

Comment:
Actually it was mentioned further down we would be adding a new metric to capture changeownership time. That may be a better indicator?

Reply:
Yes, but it does not exist yet and since I am not sure what it will be called when implemented, I am bit hesitant to add that for service level indicators. Can I update this section when I do add the metric?

Reply:
Sure maybe just for this section say that we're adding new metrics and those could be used here.

Reply:
done